src/features/jobs/jobs.py

```diff
@@ -1,50 +1,37 @@
 from fastapi import APIRouter
 from starlette.responses import JSONResponse, PlainTextResponse
 
-from config import config
 from features.jobs.use_cases.delete_job import delete_job_use_case
 from features.jobs.use_cases.get_result_job import (
     GetJobResultResponse,
     get_job_result_use_case,
 )
 from features.jobs.use_cases.start_job import StartJobResponse, start_job_use_case
 from features.jobs.use_cases.status_job import StatusJobResponse, status_job_use_case
-from restful.responses import create_response, responses
+from restful.responses import create_response
```

Comment:
didn't quite understand why responses from responses.py are not used anymore in the endpoints